.github/workflows/test.yml

```diff
@@ -26,182 +26,209 @@ jobs:
         working-directory: ./cdk
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.9
-    - name: Setup nodejs
-      uses: actions/setup-node@v2
-      with:
-        node-version: 16
-    - name: Install dependencies
-      run: |
-        pip install -r requirements.txt
-        pip install awscli build
-    - name: Install aws-cdk
-      run: npm install -g aws-cdk@$(pip freeze | grep aws-cdk.core | sed -e 's/.*==//')
-    - name: Lint with flake8/black/isort
-      run: |
-        set -x
-        export FILES=(*.py domino_cdk tests ../convert/)
-        for f in ${FILES[@]}; do
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 $f --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings.
-        flake8 $f --count --ignore=E501,W503 --exit-zero --statistics
-        black $f --check
-        isort $f --check
-        done
-    - name: Test with pytest
-      run: |
-        coverage run -m pytest tests
-    - name: Coverage report
-      run: |
-        coverage report
+      - uses: actions/checkout@v2
```

Comment:
Unless there's a part where the yaml is actually poorly formed, I'd rather not reformat all this. The dash positioning is arbitrary, while the diff obscures what the actual changes are.